src/components/Books.jsx

```diff
@@ -0,0 +1,36 @@
+import BookForm from './BookForm';
+import Buttons from './Buttons';
+
+const books = [
+  {
+    id: 1,
+    name: 'The Hunger Games',
+    writer: 'Suzanne Collins',
+    category: 'Action',
+    completed: 'not completed',
+  },
+  {
+    id: 2, name: 'Dune', writer: 'Frank Herbert', category: 'Science Fiction', completed: 'not completed',
+  },
+];
+const Books = () => (
+  <div>
+    <ul>
+      {books.map((book) => (
+        <li key={book.id}>
+          {book.category}
```

Comment:
* Kindly make your components reusable as stated in the project requirement [Check here ](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps.md#:~:text=The%20building%20blocks%20of%20your%20app%20should%20be%20set%20as%20re-usable%20components.) for example if you want to render 50 books with your current code structure that would be difficult an easy way to achieve this is by making the component `Books.jsx` reusable by making it render books and buttons only and take book data as props, you can then create another component that imports `Books.jsx` and render it as many times as you want passing in the appropriate props.